src/features/dashboard/components/Modals/AppRegisterSuccessModal/index.tsx

```diff
@@ -39,19 +39,25 @@ export const AppRegisterSuccessModal = ({
       <div className='modal__icon' style={{ background: 'var(--core-color-solid-green-100)' }}>
         <StandaloneCircleCheckRegularIcon fill='#007A22' iconSize='2xl' />
       </div>
-      <div className='modal__text'>
-        <h3 className='modal__header'>Application registered successfully!</h3>
-        <div className='modal__content' style={{ textAlign: 'left', padding: '0px 0px' }}>
-          <span>
-            Ready to take the next step?
-            <br></br>Optimise your app&apos;s capabilities by:
-            <ul>
-              <li>Creating an API token to use with your application.</li>
-              <li>Adding OAuth authentication in your app.</li>
-              <li>Selecting the scopes of OAuth authorisation for your app.</li>
-            </ul>
-            <div>Note: You can make these changes later through the dashboard.</div>
-          </span>
+      <div className='modal__content'>
+        <Heading.H4>Application registered successfully!</Heading.H4>
```

Comment:
@agrim-deriv  We don't need translation here in this component ?

Reply:
done